docs/advanced/document-store/qdrant.md

```diff
@@ -206,3 +209,58 @@ Embeddings Nearest Neighbours with "price" at most 7:
 	embedding=[5. 5. 5.],	 price=5
 	embedding=[4. 4. 4.],	 price=4
 ```
+### Example of `.filter` with a filter
+Consider Documents have the tag `price` with a value of `i`. We can create these with the following code:
```

Comment:
I think we should announce the purpose of this section before proceeding with creating documents
@NicholasDunham  what do you think ?

Reply:
I agree, as a reader the purpose isn't clear to me from what's currently here.

Reply:
Added more announcement